src/main/scala/nl.knaw.dans.dccd.convert/DccdConvertExportApp.scala

```diff
@@ -15,10 +15,216 @@
+  var dirList: List[String] = _
+  var fileList: List[File] = _
+  var file2: File = _
+
+  var metadata: Elem = _
+  var parsedMetadata: NodeSeq = _
+  var user: Elem = _
+  var parsedUser: NodeSeq = _
+
+  var dataset: String = _
+  var dcTitle: String = _
+  var depositorId: String = _
+  var dcxCreatorOrganization: String = _
+
+  val Format = new SimpleDateFormat("yyyy-MM-dd")
+
+  var date: String = _
+  var parsedDate: AnyRef = _
+  var ddmCreated: String = _
+
+  var dctRightsHolder: String = _
+  var dcDescription: String = _
+  var dcSubject: String = _
+  var dcElementType: String = _
+  var dcObjectType: String = _
+  var dcTypeMetadata: String = _
+  var dcCategory: String = _
+  var dcTaxon: String = _
+
+  var range: String = _
+
+  var dctTemporal: String = _
+
+  var dcxSpatialX: String = _
+  var dcxSpatialY: String = _
+
+  var dcIdentifierSid: String = _
+  var dcIdentifierIdentifier: String = _
+
+  var dcxRelationQualifier: String = _
+  var dcxRelationTitle: String = _
+  var dcxRelationLink: String = _
+
+  var dcxRelationQualifier_2: String = _
+  var dcxRelationTitle_2: String = _
+
+  var dcType: String = _
+
+  var language: String = _
+  var isoDefault: Locale = _
+  var languageISO3: String = _
+
+  var ddmAccessRights: String = _
+  var now: Date = _
+  var ddmAvailable: String = _
+  var ddmAudience: String = _
+
+
+  def createCsvReportFromXmlFiles(dir: String): String = {
+    dirPath = getPath(dir)
+    dirList = getListOfSubDirectories(getPath(dir).toString).toList
+    dirList.foreach { i =>
+      dirr = dirPath.toString + "/" + i + "/administrative/project_metadata.xml"
+      dirr2 = dirPath.toString + "/" + i + "/administrative/user.xml"
+      println(dirr)
+      metadata = XML.loadFile(dirr.toString)
+      parsedMetadata = parseNoWS(metadata.toString())
+      user = XML.loadFile(dirr2.toString)
+      parsedUser = parseNoWS(user.toString())
+
+      dataset = dirr
+      dcTitle = (parsedMetadata \\ "title").text
+      depositorId = (parsedUser \\ "id").text
+      dcxCreatorOrganization = (parsedMetadata \\ "ownerOrganizationId").text
+
+      date = (parsedMetadata \\ "stateChanged").text
+      parsedDate = Format.parseObject(date)
+      ddmCreated = Format.format(parsedDate)
+
+      dctRightsHolder = dcxCreatorOrganization
+      dcDescription = "Dendrochronological project"
+      dcSubject = "Dendrochronology"
+      dcElementType = "dccd_element_type: " + (parsedMetadata \\ "elementType").text
+      dcObjectType = "dccd_object_type: " + (parsedMetadata \\ "objectType").text
+      dcTypeMetadata = "dccd_type: " + (parsedMetadata \\ "type").text
+      dcCategory = "dccd_category: " + (parsedMetadata \\ "category").text
+      dcTaxon = "dccd_taxon: " + (parsedMetadata \\ "taxon").text
+
+      def timeRange(parsedData: NodeSeq): String = {
+        val parsedData = parsedMetadata
+        val first = (parsedData \\ "firstYear").text
+        val second = (parsedData \\ "lastYear").text
```

Comment:
for consistency name it last instead of second. If you don't like the dccd naming, then use start and end.